### PR TITLE
Clarifying Search Center URL

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2530

Thanks @tklarsen you are welcome to review :)

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/differences-classic-modern-search.md [hunk 1]
```diff
@@ -36,6 +36,6 @@ You use the SharePoint admin center to manage classic search and the Microsoft 3
     - Company name extraction (to be deprecated as of November 15th, 2019)
 
 - The modern search experience only shows results from the default result source. If you change the default [result source](manage-result-sources.md), both search experiences are impacted.
-- Depending on the search scenario, some Microsoft Search features might not work if the [classic global Search Center URL](https://docs.microsoft.com/sharepoint/specify-default-search-center) is not set to "yourcompanyname.sharepoint.com/search/pages". Ensure that the Search Center site collection exists, that its URL is set to "yourcompanyname.sharepoint.com/search/pages", and that all users have read access to it.
+- Depending on the search scenario, some Microsoft Search features might not work if the [classic global Search Center URL](https://docs.microsoft.com/sharepoint/specify-default-search-center) is not set to point to the URL of the default classic Search Center. Depending on your tenant, this URL is "yourcompanyname.sharepoint.com/search" or "yourcompanyname.sharepoint.com/search/pages". Further, ensure that the Search Center site collection exists and that all users have read access to it.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/differences-classic-modern-search.md:39]:
```suggestion
- Depending on the search scenario, some Microsoft Search features might not work if the [classic global Search Center URL](https://docs.microsoft.com/sharepoint/specify-default-search-center) is not set to point to the URL of the default classic Search Center. Depending on your tenant, this URL is "yourcompanyname.sharepoint.com/search" or "yourcompanyname.sharepoint.com/search/pages". Furthermore, ensure that the Search Center site collection exists and that all users have read access to it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**